### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by TheoBoucher38**

### CODE REVIEW

File: config/db.php.dist [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-define('APP_DB_USER', 'user');
-define('APP_DB_PASSWORD', 'password');
+define('APP_DB_USER', 'root');
+define('APP_DB_PASSWORD', 'root');
 define('APP_DB_HOST', 'host');
-define('APP_DB_NAME', 'name');
\ No newline at end of file
+define('APP_DB_NAME', 'projet2');
```
Comment (aldaitzwild) [inline config/db.php.dist]: No you shouldn't have to modify this file. Go back to its previous state

File: src/View/Home/homepage.html.twig [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+	<div class="mobilebodybackground d-lg-none">
+		<div class=" mobileContainer text-center my-auto mx-auto d-lg-none">
+			<img class="logorezisehomepage" src="/assets/images/gorilla.png">
```
Comment (aldaitzwild) [inline src/View/Home/homepage.html.twig]: No your class should be in CamelCase. Change it for all of them

File: src/View/Login/loginpage.html.twig [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{% extends 'layout.html.twig' %}
+
+
+{% block content %}
+    <div class="text-center">
+        <form>
+            <div class="form-group">
+                <label for="InputEmail1">Adresse Email</label>
```
Comment (aldaitzwild) [inline src/View/Login/loginpage.html.twig]: The for and the id of the input doesn't match

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
You also had to Delete your .DS_store files !

**Comment by TheoBoucher38** [comment]
Closed pull request to open another

**Pull request closed without merging**